go.mod

```diff
@@ -17,23 +17,25 @@ replace (
 
 require (
```

Comment:
one of these change broke the build

Reply:
There is some k8s error in the vendor virtctl's script, I'm trying to fix it.